include/mpsl/mpsl_assert.rst

```diff
@@ -0,0 +1,21 @@
+.. _mpsl_assert:
+
+Multiprotocol Service Layer assert
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
```

Comment:
Can you start with an intro on what this library is for and why you would use it?
Something like:

The Multiprotocol Service Layer assert library makes it possible to add a custom assert handler to the :ref:`nrfxlib:mpsl` library.
You can then use this assert handler to ...? (print custom error messages? recover from errors?)

Reply:
updated